index.html

```diff
@@ -26,5 +26,139 @@ <h3 class="profession">I’m a Software Developer</h3>
+                    </ul>
+                    <button class="project2-button">See Project</button>   
+            </div>
+            <div class="project2">
+                <h1 class="project2-title">Profesional Art Printing Data</h1>
+                <p class="project2-description">A daily selection of privately personalized reads; no accounts or
+                    sign-ups required. has been the industry's standard</p>
+                    <ul class="project2-tech">
+                        <li><button>html</button></li>
+                        <li><button>bootstrap</button></li>
+                        <li><button>Ruby</button></li>
+                    </ul>
+                    <button class="project2-button">See Project</button>   
+            </div>
+            <div class="project2">
+                <h1 class="project2-title">Profesional Art Printing Data</h1>
+                <p class="project2-description">A daily selection of privately personalized reads; no accounts or
+                    sign-ups required. has been the industry's standard</p>
+                    <ul class="project2-tech">
+                        <li><button>html</button></li>
+                        <li><button>bootstrap</button></li>
+                        <li><button>Ruby</button></li>
+                    </ul>
+                    <button class="project2-button">See Project</button>   
+            </div>
+        </div>          
+    </section>
+    <section id="about-section">
+        <div class="about-title">
+            <h1>About me</h1>
+        </div>
+        <p class="about-description">Hello I’m a software developer! I can help you build a product , feature or website Look through some of my work and experience! If you like what you see and have a project you need coded, don’t hestiate to contact me.</p>
+        <button class="button-resume">Get My Resume</button>
+        <div class="about-line"></div>
+        <div class="language">
+            <div class="language-word">
+                <img src="./Rectangle 55.png" alt="rectangle icon">
+                <h3>Language</h3>
+            </div>
+            <ul class="language-list">
+                <li>Javascript</li>
+                <li>Ruby</li>
+                <li>Html</li>
+                <li>Css</li>
+            </ul>
+        </div>
+        <div class="frameworks">
+            <div class="frameworks-word">
+                <img src="./Rectangle 56.png" alt="rectangle icon">
+                <h3>Frameworks</h3>
+            </div>
+            <ul class="frameworks-list">
+                <li>Bootstrap</li>
+                <li>Ruby on Rails</li>
+                <li>RSPec</li>
+                <li>CapyBara</li>
+                <li>Selenium</li>
+            </ul>
+        </div>
+        <div class="skills">
+            <div class="skills-word">
+                <img src="./Rectangle 57.png" alt="rectangle icon">
+                <h3>Skills</h3>
```

Comment:
- [ ] nice job. please kindly centralize the skills heading so it follows the original design. 
yours
![Screenshot (746)_LI](https://user-images.githubusercontent.com/65298048/212430706-e32a3be2-3280-4813-8ab0-a8aba9c833ba.jpg)
figma
![Screenshot (747)](https://user-images.githubusercontent.com/65298048/212430786-bce4f5c0-ed76-406d-b646-3baa52cab189.png)